multipass/reference/multipass-mount-command.md

```diff
@@ -6,7 +6,12 @@ Please don't copy it blindly, first update the URL passed to the form with the c
 
 > See also: [Mount](/t/28470), [How to share data with an instance](/t/27189), [`umount`](/t/27214), [ID mapping](/t/45986).
 
-The `multipass mount` command maps a local directory from the host to an instance, with the possibility to specify the [mount](/t/28470) type (classic or native) and define group or user [ID mappings](/t/45986).
+The `multipass mount` command maps a local directory from the host to an instance, with the possibility to specify the [mount](/t/28470) type (classic or native) and define group or user [ID mappings](/t/45986). Here's the syntax for mapping a local directory to your virtual machine using the classic mount:
+
+```shell
+multipass mount --type=classic /host/path instance:/instance/path
+```
+The `classic` mounts use technology built into multipass and allows for higher compatibility. However, it sacrifices performance. The `native` mounts on the other hand use hypervisor and/or platform specific mounts to offer better performance but has [limited compatibility](https://multipass.run/docs/mount#native-mounts).
```

Comment:
Since we also have a [Mount](https://multipass.run/docs/mount) reference page that explains the difference between classic and native mounts, I think it would be best to add any additional info (such as the comparison you added below) to that page directly. 

In this case, I think it would make sense to add a link to the [How to share data with an instance](https://multipass.run/docs/share-data-with-an-instance) guide, where users can find more hands-on examples on how to use the `multipass mount` command. What do you think? This guide is actually linked at the top of the page, but I'll suggest a paragraph to highlight it here in the flow of the text instead.

```suggestion
See also [How to share data with an instance](/t/27189) for examples of how you can use the `multipass mount` command to share data between your host and an instance.
```

Reply:
I think that would be okay. With this addition the comparsion below becomes obsolete right? should we remove it completely?

Reply:
Yes, if you move the additional info to the a [Mount](https://multipass.run/docs/mount) page there is no need to repeat it here, a link is enough (and I believe we're already linking it at the top). What do you think?

Reply:
Yes, that works